core/templates/dev/head/domain/utilities/browser-checker.service.ts

```diff
@@ -26,21 +26,19 @@ const CONSTANTS = require('constants.ts');
 })
 export class BrowserCheckerService {
   private _supportsSpeechSynthesis(): boolean {
-    var supportLang = false;
     if (window.hasOwnProperty('speechSynthesis')) {
-      // TODO(#7165): Replace any with correct type.
-      speechSynthesis.getVoices().forEach((voice: any) => {
+      speechSynthesis.getVoices().some((voice: SpeechSynthesisVoice) => {
```

Comment:
You'll need a "return" before this statement, otherwise it will not do anything and this function will always return false.

Reply:
done!